platform/solr/platform-solr-server-standalone/src/main/resources/solr/conf/schema_metacard.xml

```diff
@@ -566,11 +576,11 @@
           required="false"/>
    <field name="id_txt_tokenized_has_case" type="text_general_has_case" indexed="true"
           stored="false" required="false"/>
-   
-   <field name="metacard_type_name" type="string" indexed="true" stored="true" required="true"/>
-   
-   <field name="metacard_type_object" type="binary" indexed="false" stored="true" required="true"/>
+     <!-- OMIT
+        <field name="metacard_type_name" type="string" indexed="true" stored="true" required="true"/>
```

Comment:
Looks like these fields have been dead for a long time.  I would suggest removing these completely.